From 770b5acf1736297669d000d1f673ba287ef1d95e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Diego=20Elio=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Wed, 2 Feb 2011 14:20:00 +0100
Subject: [PATCH] Skip blankslate tests even when using minitest.

At least with Ruby 1.9.2's minitest, simply defining the suite method is
not going to be enough to skip over these tests; instead use the skip
method if it's available.
---
 test/test_blankslate.rb |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/test/test_blankslate.rb b/test/test_blankslate.rb
index a46eeb6..222b9bc 100644
--- a/test/test_blankslate.rb
+++ b/test/test_blankslate.rb
@@ -89,6 +89,10 @@ class TestBlankSlate < Test::Unit::TestCase
   end
 
   def setup
+    if Object::const_defined?(:BasicObject) and respond_to?(:skip)
+      skip "BlankSlate is not used in this environment" 
+    end
+
     @bs = BlankSlate.new
   end
 
-- 
1.7.4

